src/System.Net.Mime/src/System.Net.Mime.csproj

```diff
@@ -1,73 +0,0 @@
-    <Compile Include="System\Net\Mime\Base64WriteStateInfo.cs" />
-    <Compile Include="System\Net\Mime\QuotedPrintableStream.cs" />
-    <Compile Include="System\Net\Mime\CloseableStream.cs" />
-    <Compile Include="System\Net\Mime\EightBitStream.cs" />
-    <Compile Include="System\Net\Mime\EncodedStreamFactory.cs" />
-    <Compile Include="System\Net\Mime\IEncodableStream.cs" />
-    <Compile Include="System\Net\Mime\QEncodedStream.cs" />
-    <Compile Include="System\Net\Mime\WriteStateInfoBase.cs" />
-    <Compile Include="System\Net\Mime\BaseWriter.cs" />
-    <Compile Include="System\Net\Mime\TransferEncoding.cs" />
-    <Compile Include="System\Net\Mime\ContentDisposition.cs" />
-    <Compile Include="System\Net\Mime\ContentType.cs" />
-    <Compile Include="System\Net\Mime\DispositionTypeNames.cs" />
-    <Compile Include="System\Net\Mime\HeaderCollection.cs" />
-    <Compile Include="System\Net\Mime\MediaTypeNames.cs" />
-    <Compile Include="System\Net\Mime\MimeBasePart.cs" />
-    <Compile Include="System\Net\Mime\SmtpDateTime.cs" />
-    <Compile Include="System\Net\Mime\MultiAsyncResult.cs" />
-    <Compile Include="System\Net\Mime\TrackingStringDictionary.cs" />
-    <Compile Include="System\Net\Mime\TrackingValidationObjectDictionary.cs" />
-    <Compile Include="System\Net\Mail\MailHeaderID.cs" />
-    <Compile Include="System\Net\Mail\MailHeaderInfo.cs" />
-    <Compile Include="System\Net\Mail\BufferBuilder.cs" />
-    <Compile Include="$(CommonPath)\System\IO\DelegatingStream.cs">
-      <Link>Common\System\IO\DelegatingStream.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\InternalException.cs">
-      <Link>Common\System\Net\InternalException.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\LazyAsyncResult.cs">
-      <Link>Common\System\Net\LazyAsyncResult.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Mail\MailBnfHelper.cs">
-      <Link>Common\System\Net\Mail\MailBnfHelper.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Logging\LoggingHash.cs">
-      <Link>Common\System\Net\Logging\LoggingHash.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Logging\GlobalLog.cs">
-      <Link>Common\System\Net\Logging\GlobalLog.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Logging\EventSourceLogging.cs">
-      <Link>Common\System\Net\Logging\EventSourceLogging.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Shims\DBNull.cs">
-      <Link>Common\System\Net\Shims\DBNull.cs</Link>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net463'">
-    <TargetingPackReference Include="System" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
```

Comment:
Will address the nit in a separate PR, as triggering outerloop CI again for this change does not add value.